src/models/common.ts

```diff
@@ -8,5 +8,6 @@ export interface Dictionary {
 }
 export interface Row {
   color: COLOR;
+  bottomBorder: boolean;
```

Comment:
bottomBorder is also(more) suitable as general property of Table

In general if a table has new line separated entries it should by default show row separator for each row.
If mentioned as general property by user while creating new Table(props) then it should also be shown.

Reply:
Sounds good.

Just to be sure, if I do new Table(props) and say "rowSeparator: true" but then I addRow(..., { rowSeparator: false }) I guess that addRow should have priority correct?

I'd like to continue to support having only some rows with separator to support manual grouping of rows:
![image](https://user-images.githubusercontent.com/5521715/121142507-b8e0be80-c844-11eb-9c0e-d4860f01ab52.png)